client/src/modules/account_statement/account_statement.html

```diff
@@ -78,7 +94,8 @@
           ui-grid-resize-columns
           ui-grid-move-columns
```

Comment:
Do we need `ui-grid-move-columns`?

Reply:
I think it's helpful if a user wants to customize their view.  In the long term, we'll make our PDFs print out in the same order as the ui-grid, and that will give users a lot of power to customize their application.